lib/matplotlib/projections/geo.py

```diff
@@ -190,7 +190,7 @@ def set_longitude_grid(self, degrees):
         """
         Set the number of degrees between each longitude grid.
         """
-        number = (360.0 / degrees) + 1
+        number = int(360.0 / degrees) + 1
```

Comment:
I think we should always use numpy stuff when we can. We depend on numpy, so there is no reason to not use it (unless I am missing something).